impl/src/main/resources/META-INF/WebConfigParamsLogger.vm

```diff
@@ -34,88 +34,88 @@ public class WebConfigParamsLogger
+            String[] expectedValues = StringUtils.trim(StringUtils.splitShortString("$webConfigParam.expectedValues",','));
+            for (int i = 0; i < expectedValues.length; i++)
             {
-                boolean found = false;
-                String[] expectedValues = StringUtils.trim(StringUtils.splitShortString("$webConfigParam.expectedValues",','));
-                for (int i = 0; i < expectedValues.length; i++)
-                {
 #if ($webConfigParam.isIgnoreUpperLowerCase())
-                    if (paramValue.equalsIgnoreCase(expectedValues[i]))
+                if (paramValue.equalsIgnoreCase(expectedValues[i]))
 #else
-                    if (paramValue.equals(expectedValues[i]))
+                if (paramValue.equals(expectedValues[i]))
 #end
-                    {
-                        found = true;
-                        break;
-                    }
-                }
-                if (!found)
                 {
-                    if (log.isLoggable(Level.WARNING))
-                    {
-                        log.warning("Wrong value in context init parameter '$webConfigParam.name' (='" + paramValue + "'), using default value #if($webConfigParam.defaultValue)'$webConfigParam.defaultValue'#else'null'#end");
-                    }
+                    found = true;
+                    break;
                 }
             }
-#elseif ($webConfigParam.classType)
-            else
+            if (!found)
             {
-                try
+                if (log.isLoggable(Level.WARNING))
                 {
-                    ${webConfigParam.classType}.valueOf(paramValue);
+                    log.warning("Wrong value in context init parameter '$webConfigParam.name' (='" + paramValue + "'), using default value #if($webConfigParam.defaultValue)'$webConfigParam.defaultValue'#else'null'#end");
                 }
-                catch(Exception e)
+            }
+            else
+            {
+                logMessageToAppropriateLevel("Init context parameter found '$webConfigParam.name' set to '" + paramValue + "'");
+            }
+        }
+#elseif ($webConfigParam.classType)
+        else
+        {
+            try
+            {
+                ${webConfigParam.classType}.valueOf(paramValue);
+            }
+            catch(Exception e)
+            {
+                if (log.isLoggable(Level.WARNING))
                 {
-                    if (log.isLoggable(Level.WARNING))
-                    {
-                        log.warning("Wrong value in context init parameter '$webConfigParam.name' (='" + paramValue + "'), using default value #if($webConfigParam.defaultValue)'$webConfigParam.defaultValue'#else'null'#end");
-                    }
+                    log.warning("Wrong value in context init parameter '$webConfigParam.name' (='" + paramValue + "'), using default value #if($webConfigParam.defaultValue)'$webConfigParam.defaultValue'#else'null'#end");
```

Comment:
Majority of the "changes" are just me indenting this to the left once since I removed an if statement this code was nested in.